test/remote-test.js

```diff
@@ -1873,3 +1960,499 @@ describe('Remote', function() {
     });
   });
 });
+
+describe.skip('Request API consistency tests', function() {
```

Comment:
Do we have an issue open with eslint about spreads? Are we using the latest version of eslint ?